cypress/e2e/dashboard.cy.ts

```diff
@@ -0,0 +1,36 @@
+function resetToDefaultLayout() {
```

Comment:
This function already exists in this repository. Please add it as a custom command instead of copy pasting it.

Reply:
Fixed